vllm_spyre/model_executor/model_loader/spyre.py

```
@@ -127,13 +127,12 @@ def load_weights(self, model_config: ModelConfig, max_prompt_length: int,
             sys.path.append("/home/senuser/aiu-fms")
 
             if envs_spyre.VLLM_SPYRE_DYNAMO_BACKEND == "sendnn_decoder":
-                from aiu_as_addon import aiu_adapter, aiu_linear  # noqa: F401
+                from fms_mo.aiu_addons.gptq import (  # noqa: F401
+                    gptq_aiu_adapter, gptq_aiu_linear)
                 linear_type = "gptq_aiu"
-                logger.info("Loaded `aiu_as_addon` functionalities")
+                logger.info("Loaded `aiu_addons` functionalities")
             else:
-                from cpu_addon import cpu_linear  # noqa: F401
                 linear_type = "gptq_cpu"
```

Should we perhaps just raise an exception here (since we don't expect it to work)?

---

I inserted a logger warning. This way it does not directly raise an exception if this should work in the future due to some changes in the fms repos.

---

If it doesn't work now, I would still prefer to raise an exception rather than warning if it is unsupported configuration.  Exception is much better feedback to the user.